### PR TITLE
Web development task3 done

### PR DESCRIPTION
**Pull request by piyush-bit**
Went towards a fancy design and it looks pretty good 

but due to increased complexity was not able to make it that responsive 

## Hope you like it

### CODE REVIEW

**Comment by Khitishgithub** [comment]
Your page is looking good in desktop view . make it responsive for smaller screen devices .

**Comment by piyush-bit** [comment]
@Khitishgithub is it okay now 
😀

**Comment by Khitishgithub** [comment]
Superb !! . Keep coding .

File: Web Development/Task/Task Submission/Piyush Kumar/links.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+live_link = https://piyush-bit.github.io/induction_portfolio/
```
Comment (JellyCone1) [inline Web Development/Task/Task Submission/Piyush Kumar/links.txt:1]: -1, cannot select text :(

Comment (piyush-bit) [reply Web Development/Task/Task Submission/Piyush Kumar/links.txt:1]: that isn't a bug 
it was a trade-off for that yellow circle behind the circle 

did I make the right decision

Comment (Khitishgithub) [reply Web Development/Task/Task Submission/Piyush Kumar/links.txt:1]: It looks nice .

**Pull request merged**